src/odemis/acq/stitching/__init__.py

```diff
@@ -16,6 +16,62 @@
 '''
 from __future__ import division
 
-
 from ._registrar import *
 from ._weaver import *
+
+import copy
+
+
+def register(tiles, method="REGISTER_SHIFT"):
+    """ 
+    tiles (list of DataArray of shape YX or tuples of DataArrays): The tiles to compute the registration. 
+    If it's tuples, the first tile of each tuple is the “main tile”, and the following ones are 
+    dependent tiles.
+    method (REGISTER_*): REGISTER_SHIFT → ShiftRegistrar, REGISTER_IDENTITY → IdentityRegistrar
+    returns:
+        tiles (list of DataArray of shape YX or tuples of DataArrays): The tiles as passed, but with updated 
+        MD_POS metadata
+    """
+    
+    if method == "REGISTER_SHIFT":
+        registrar = ShiftRegistrar()
+        
+    elif method == "REGISTER_IDENTITY":
+        registrar = IdentityRegistrar()
+    
+    else:
+        raise ValueError("Invalid registrar") 
```

Comment:
Include the content of ```method```  (with %s) into the ValueError message. This way, if this actually happens, there is some hope to be able to debug it immediately.